tests/docker.NOAA_GEFS.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<pecan>
+  <outdir>/data/tests/sipnet</outdir>
+  <database>
+    <bety>
+      <driver>PostgreSQL</driver>
+      <user>bety</user>
+      <password>bety</password>
+      <host>postgres</host>
+      <dbname>bety</dbname>
+      <write>FALSE</write>
+    </bety>
+  </database>
+  <model>
+    <type>SIPNET</type>
+  </model>
+  <run>
+    <site>
+      <id>676</id>
+    </site>
+    <inputs>
+      <met>
+        <source>NOAA_GEFS</source>
+      </met>
+    </inputs>
+    <start.date>2011-01-01</start.date>
+    <end.date>2011-12-31</end.date>
```

Comment:
Dates work differently here because it's a forecast product -- start date needs to be less than 4 days before run time, and end date not more than 35 days after that. I think we discussed how to implement these variable dates at one point, but I forget what we concluded.

(I realize this is going to be hard to test until we get the GEFS download function working again. I've made some progress there but am not ready to open the PR yet)

Reply:
See below for my idea of how to handle this inside test.R